openmapflow/notebooks/compare_ceo_labels.ipynb

```diff
@@ -0,0 +1,2563 @@
+{
```

Comment:
<p>Can the notebook outputs be cleared?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/openmapflow/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>